### ISSUE TITLE
Provide API for an issue list 

### ISSUE THREAD

**Issue by kevincianfarini** [climatechangemakers/climate-changemakers-act#31]
This issue list should read from the database and specify a focus issue as well as a list of other issues. The API might look like the following. 

```js
// GET /api/issues

{
  focusIssue: {
    title: "string",
    talkingPoints: [
      { title: "string", content: "string" },
      ...
    ]
  }, 
  otherIssues: [ ... ]
}
```

Alternatively, we could also store _all_ of the content of an `Issue` as markdown text. That could be rendered on the frontend into HTML. A quick search on how to render markdown to HTML yield [this guide](https://blog.logrocket.com/how-to-safely-render-markdown-using-react-markdown/). 

```js
{
  focusIssue: "string", // markdown compliant formatting  
  otherIssues: [ ... ]
}
```

@jacknugent would it be possible to take things like `<h1>` elements and render those as collapsible sections like in the mockup pictured below? 

![image](https://user-images.githubusercontent.com/13242326/130306133-5d65e45a-915e-4e61-8886-85e863c6926e.png)

If the level of effort to render content contained within a certain header as collapsible is manageable, then I would prefer to store the issue as raw markdown. It makes the process simpler when we start integrating a CMS later on.

**Comment by kevincianfarini** [comment]
Thinking about this more, I think regardless of the avenue we take we're going to end up rendering markdown. The contents within an issue section is never going to be a wall of text. It will contain bullets, references (links), italics, bolding, etc.

**Comment by jacknugent** [comment]
I would go with the first `GET` solution, where it's an array of issues. Parsing HTML for a particular tag sounds messy. It would also be harder to sort/modify issues in a database if the title of the issue is embedded in an html string.

We can render markdown no problem, it has [built-in support](https://reactjs.org/docs/dom-elements.html):

```javascript
function createMarkup() {
  return {__html: 'First &middot; Second'};
}

function MyComponent() {
  return <div dangerouslySetInnerHTML={createMarkup()} />;
}
```

But why are we building out this endpoint ourselves and not using a CMS? With a CMS, we would have access to a GUI that would build out an API like this one for us, and the GUI would let anybody, developer or non-developer, edit in a WYSIWG editor.

**Comment by kevincianfarini** [comment]
This endpoint is a stop gap between now and when we have resources to roll out a CMS. This action tool is top priority right now.

**Comment by kevincianfarini** [comment]
The long term landscape of things looks like the following

1. Roll out the action tool with manually managed issues in the DB table (Version 1) 
2. Evaluate a CMS solution 
3. Implement the CMS and migrate database values to whatever new schema we adopt 
4. Migrate the necessary bits of the action tool to take advantage of the CMS data (Version 2)

**Comment by kevincianfarini** [comment]
The API the frontend consumes should remain stable through this migration.

**Comment by kevincianfarini** [comment]
Closed via #33. Follow up in #50
